src/main/external-resources/renderers/LG-WebOS.conf

```diff
@@ -15,7 +15,7 @@ RendererIcon = lg-lb6500.png
 # ============================================================================
 #
 
-UserAgentSearch = LGE WebOS TV|LG WebOSTV
+UserAgentSearch = LG. WebOS TV|LG WebOSTV
```

Comment:
@SubJunk ```.``` matches all, so it will also match ```E```. Any string matching the old will match the new AFAICS.